### ISSUE TITLE
Attribute has an unknown type: Property" error on Batch Upsert

### ISSUE THREAD

**Issue by jason-fox** [stellio-hub/stellio-context-broker#1136]
Stellio 2.12.2 when performing an upsert using the following [@context file](https://raw.githubusercontent.com/FIWARE/tutorials.Time-Series-Data/NGSI-LD/data-models/ngsi-context.jsonld)

```console
curl -L 'http://localhost:1026/ngsi-ld/v1/entityOperations/upsert?options=update' \
-H 'Content-Type: application/json' \
-H 'Link: <http://context/ngsi-context.jsonld>; rel="http://www.w3.org/ns/json-ld#context"; type="application/ld+json"' \
-d '[
  {
    "id": "urn:ngsi-ld:TemperatureSensor:003",
    "type": "TemperatureSensor",
    "category": {
      "type": "Property",
      "value": [
        "actuator",
        "sensor"
      ]
    }
  },
  {
    "id": "urn:ngsi-ld:TemperatureSensor:004",
    "type": "TemperatureSensor",
    "category": {
      "type": "Property",
      "value": [
        "actuator",
        "sensor"
      ]
    }
  }
]'
```

Results in:

```json
{
    "success": [],
    "errors": [
        {
            "entityId": "urn:ngsi-ld:TemperatureSensor:003",
            "error": [
                "Attribute https://uri.fiware.org/ns/data-models#category has an unknown type: Property"
            ]
        },
        {
            "entityId": "urn:ngsi-ld:TemperatureSensor:004",
            "error": [
                "Attribute https://uri.fiware.org/ns/data-models#category has an unknown type: Property"
            ]
        }
    ]
}
```

**Comment by bobeal** [comment]
Good catch. If not present, the core context is not added in the batch operations (and the `Property` term is not correctly expanded. PR in progress.

**Comment by bobeal** [comment]
Just released a 2.12.3 with the fix.
